.changeset/witty-pans-roll.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
nit: empty changeset is not needed is only Java file are edited.

Reply:
Yeah, I agree it's not necessary, but I sometimes create one to remind myself of what changes are pending since it creates an item in the changeset folder associated with this commit.